packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -380,3 +393,53 @@ Then('I click the submit button', () => {
     name: new RegExp(`^((submit)|(send code))$`, 'i'),
   }).click();
 });
+
+And('I confirm {string} error is accessible in password field', () => {
```

Comment:
I moved this to shared.ts so that they can be shared between all the screens with pw validation